buildSrc/src/main/kotlin/toolkit-testing.gradle.kts

```diff
@@ -0,0 +1,70 @@
+// Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import gradle.kotlin.dsl.accessors._19c5183b2e35b82afb9ef9cd50f1ef9e.java
```

Comment:
how does this work?

Reply:
Thats intellij trying to be smart when I copy paste code. It should never be there